crates/ruff_linter/src/rules/pydocstyle/snapshots/ruff_linter__rules__pydocstyle__tests__D406_sections.py.snap

```diff
@@ -1,19 +1,14 @@
 ---
 source: crates/ruff_linter/src/rules/pydocstyle/mod.rs
 ---
-sections.py:30:5: D406 [*] Section name should end with a newline ("Returns")
+sections.py:32:5: D406 [*] Section name should end with a newline ("Returns")
    |
-28 |           "('Returns', not 'Returns:')")
-29 |   def superfluous_suffix():  # noqa: D416
-30 |       """Toggle the gizmo.
-   |  _____^
-31 | | 
-32 | |     Returns:
-33 | |     -------
-34 | |     A value of some sort.
-35 | | 
-36 | |     """
-   | |_______^ D406
+30 |     """Toggle the gizmo.
+31 | 
+32 |     Returns:
+   |     ^^^^^^^ D406
+33 |     -------
```

Comment:
The range here could possibly be just the range of the invalid suffix (in this case, the colon at the end of the header), since the fix is just to delete the suffix

Reply:
I still feel like for this one, the range is _somewhat_ confusing, because we're specifically complaining about the colon at the end of "Returns:", which is actually _excluded_ from the range here? But no strong opinion :)

Reply:
Honestly, I could go either way. I left it as-is for now but I'm open to changing it.